### DISCUSSION TITLE
@Entity

### DISCUSSION THREAD

**Discussion by clojj** [knittl/ddd-annotations#6] [General]
Hi, nice annotations.

I wonder, if @Entity would make sense too ?
Ofcourse an aggregate _is_ one or more entities... not sure though

**Comment by knittl** [comment]
Hi @clojj,

thanks for your feedback. In fact, there already is an annotation for entities, but you have to use it as `@Domain.Entity`. This is to avoid conflicts with JPA/Hibernate `@Entity` annotation.

The same reasoning applies to `@Domain.Repository`: to avoid conflicts with Spring Data's `@Repository` annotation, which even mentions DDD in its JavaDoc.

**Reply by clojj** [reply]
oh nice! Sorry, didn't see that by flying over this repo.
